spec/requests/api/members_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe "api/members" do
+
+  def json
+    JSON.parse(response.body)
+  end
+
+  describe 'POST api/members' do
+
+    it 'creates a new member' do
+      post api_member_path(:email => "newbie@test.org")
```

Comment:
the `post#create` path is plural... `api_members_path`. The email is posted in the body of the request, not in the URL.

```ruby
post api_members_path, { email: "newbie@test.org" }
```